### PR TITLE
Add a bit of interlinking in the QFC pages

### PR DESCRIPTION
**Pull request by suricactus**

### CODE REVIEW

File: documentation/get-started/tutorials/advanced-setup-qfc.en.md [hunk 1]
```diff
@@ -61,3 +61,5 @@ It requires your database to be publicly accessible and credentials must be save
 
 !!! note
     When using `offline editing`, QField will work on a local copy of the database in a GeoPackage, which will be synced by QFieldCloud to the original database. This is the best choice if the connection in the field is not reliable. Changes will only be visible to users once they sync to QFieldCloud. As a local copy is created, advanced PostGIS features will not be available on QField. Just like for regular GeoPackages, if you define relationships, it is recommended to use UUIDs instead of integer primary keys to avoid conflicts if multiple users create data at the same time.
+
+You can find more information on [QFieldCloud technical reference](../../reference/qfieldcloud/concepts.en.md).
```
Comment (m-kuhn) [inline documentation/get-started/tutorials/advanced-setup-qfc.en.md:65]: That works with translation I assume?

**Review by m-kuhn — APPROVED** [review]
Thanks

File: documentation/get-started/tutorials/advanced-setup-qfc.en.md (hunk 1 above)
Comment (suricactus) [reply documentation/get-started/tutorials/advanced-setup-qfc.en.md:65]: I don't know, we better ask @marioba @why-not-try-calmer

Comment (suricactus) [reply documentation/get-started/tutorials/advanced-setup-qfc.en.md]: Ok decided to make it without the `.en` suffix, works for sure as for the rest of the pages.

Comment (m-kuhn) [reply documentation/get-started/tutorials/advanced-setup-qfc.en.md]: Thanks!

**Pull request merged**